content/administrators/people/student-profile.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Student Profile"
+linktitle: "Student Profile"
+---
+
+The student profile is a key component of Gibbon, as it gives administrators and teachers full access to all the necessary information about an individual student.
```

Comment:
It may be handy to note here that students can see their own profile, and parents can see the profile of their children. Other users may see a brief version or full version of the profile, depending on the permissions set in Manage Permissions.